lib/rules/new-line-before-expect.js

```diff
@@ -8,14 +8,21 @@
 var hasPaddingBetweenTokens = require('../helpers/hasPaddingBetweenTokens')
 
 var blockRegexp = /^((f|x)?(it|describe))$/
+var lastExpectNode
```

Comment:
Well, actually, it would not work, because it will reset every time, even when it will be called for the second expect node.